docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -218,6 +218,10 @@ public class CounterBean {
 ** Injection point metadata
 * Scopes and contexts
 ** `@Dependent`, `@ApplicationScoped`, `@Singleton`, `@RequestScoped` and `@SessionScoped`
+** xref:transaction.adoc#transaction-scope[`@TransactionScoped`]
```

Comment:
Hmm, this section is about CDI features, i.e. features from the spec point of view and `@TransactionScoped` is a custom scope/context which is mentioned below...

Reply:
Well, this and the following section is about supported and unsupported CDI features. And the goal is to give overview of ArC and CDI compatibility, not an overview of quarkus features.

> It's not a custom scope from the point of view of users.

I think that regular users do not know what a custom scope is ;-) 

I'm not sure. If you really believe it belongs here I can live with that. What do @manovotn and @Ladicek think?

Reply:
Maybe I could at least add a note to link to https://quarkus.io/guides/cdi#what-scopes-can-i-actually-use-in-my-quarkus-application ?

Reply:
This is a bit of a gray area IMHO. Strictly speaking, @mkouba is right in that CDI doesn't define `@TransactionScoped`, so it doesn't belong here. On the other hand, `@SessionScoped` is defined by CDI and so belongs here, even though vast majority of users can't actually use it, because it's only supported with Undertow.

It seems to me that https://quarkus.io/guides/cdi#what-scopes-can-i-actually-use-in-my-quarkus-application already provides all the necessary information, so I just wouldn't list the scopes here at all, and instead add a link.

Reply:
> I just wouldn't list the scopes here at all, and instead add a link.

Thanks. I updated the PR to do just that.

Reply:
Coming a bit late to the party but I agree with Ladislav and Martin in that transactional scoped is by CDI definition a custom scope.
The updated version of your PR is IMO a good approach, so +1